sdk/communication/azure-communication-phonenumbers/README.md

```diff
@@ -16,153 +16,129 @@ Install the Azure Communication Phone Numbers client library for Python with [pi
 pip install azure-communication-phonenumbers
 ```
 
-# Key concepts
+## Key concepts
 
-## CommunicationPhoneNumberClient
 ### Initializing Phone Numbers Client
 ```python
-# You can find your endpoint and access token from your resource in the Azure Portal
```

Comment:
are we getting rid of endpoint/access token initialization?

Reply:
We have just been using the connection string in Python AFAIK.